### ISSUE TITLE
Wiki example *Writing a Language Service Plugin* not working / ?not triggered by tsc?

### ISSUE THREAD

**Issue by tomsoftware** [microsoft/TypeScript#34548]
**TypeScript Version:**  3.6.4

**Search Terms:** Example, Wiki, Docu, Writing a Language Service Plugin, Logging via `TSS_LOG`

**Code**
(https://github.com/microsoft/TypeScript/wiki/Writing-a-Language-Service-Plugin)
Code of Wiki in self contained project
(https://github.com/tomsoftware/sample-ts-plugin)

**Expected behavior:**
work as the wiki says:
- log to file
- remove expression in result `test.js`

**Actual behavior:**
- no log file is created
- nothing is changed in the output of the `test.ts` file
(it looks like the plugin is not triggered at all)

**Playground Link:** 
https://github.com/tomsoftware/sample-ts-plugin

**Comment by RyanCavanaugh** [comment]
From the title it sounds like you're expecting something to happen when `tsc` runs. From the docs https://github.com/microsoft/TypeScript/wiki/Writing-a-Language-Service-Plugin (emph. added):

> TypeScript Language Service Plugins ("plugins") are for changing the **editing experience only**. The core TypeScript language remains the same. Plugins can't add new language features such as new syntax or different typechecking behavior, and **plugins aren't loaded during normal commandline typechecking or emitting**.

**Comment by tomsoftware** [comment]
My **guess** is that the **Testing Locally** think **not working** like the Wiki says - but it is only a guess - may the wiki is just outdated or I did something wrong. And Yes I expect:
- log file is created
- the plugin will do something

> 
> Testing Locally
> 
> To locally test your plugin, set up a sample project and give it the path to your plugin in the tsconfig.json file. For example:
> 
> your_plugin/index.ts
> your_plugin/index.js (compiled by tsc)
> sample_project/tsconfig.json
> where sample_project/tsconfig.json contains
> ```
> {
>     "compilerOptions": {
>         "plugins": [{
>             "name": "../your_plugin",
>         }]
>     }
> }
> ```
